src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+    private int thirdLeg;
+    private int fourthLeg;

```

Comment:
3 sides are enough, you even don't use thirdLeg in area calculations

Reply:
++